src/main/java/com/xloop/resourceloop/authenticationservice/Controller/AuthController.java

```diff
@@ -0,0 +1,35 @@
+package com.xloop.resourceloop.authenticationservice.Controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.xloop.resourceloop.authenticationservice.JPARepository.UserRepository;
+import com.xloop.resourceloop.authenticationservice.Model.User;
+
+@RestController
+@RequestMapping("/auth")
+public class AuthController {
+    @Autowired
+    private UserRepository userRepo;
+
+    @PostMapping("/register")
+    public ResponseEntity<String> register(@RequestBody User user){
+        User exist_user = userRepo.findByEmail(user.getEmail());
+        if(exist_user != null){
+            return ResponseEntity.status(208).body("User Already Exist");
```

Comment:
@hunainparekh2211006KHICND `208` isn't the correct response code here, it's related to document management, `409` refers to a conflict and registers as a client error as it is `4XX`